front_end/migration_scripts/11_to_12_check.sql

```diff
@@ -0,0 +1,3 @@
+SELECT COUNT(*) AS count
+               FROM pragma_table_info("assignments")
```

Comment:
Again, I know this is nitpicky, and we haven't always followed this, but please indent the FROM and WHERE lines the same as the SELECT line.

```
SELECT COUNT(*) AS count
FROM pragma_table_info("assignments")
WHERE name = "allowed_ip_addresses"
```